snippets/lfs/lfs_optimized.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_FILE_SYSTEM=y
+CONFIG_FILE_SYSTEM_LITTLEFS=y
+CONFIG_FS_LITTLEFS_FMP_DEV=y
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+
+CONFIG_FS_LITTLEFS_CACHE_SIZE=512
+CONFIG_FS_LITTLEFS_READ_SIZE=256
+CONFIG_FS_LITTLEFS_PROG_SIZE=256
+CONFIG_FS_LITTLEFS_LOOKAHEAD_SIZE=128
+CONFIG_FS_LITTLEFS_FC_HEAP_SIZE=2048
```

Comment:
did these end up doing thing or are they overwritten by the dts ones. if the're over written by dt ones, can probably blow these up. if vice versa, can probably blow those up.

Reply:
I believe devicetree gets overwritten by kconfig when using diff values in the prj.conf and checking using guiconfig. Going to blast the kconfig ones since you have to enter in values in the devicetree anyways, minus heap size since that isn't specified in devicetree

Reply:
Actually that file can be blasted. Just made a second conf file to quickly swap for testing